experimental/design-ios-gradle-dsl/build.gradle.kts

```diff
@@ -0,0 +1,65 @@
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.compose")
+}
+
+kotlin {
+    iosX64()
+    iosArm64()
+}
+
+compose.ios {
+    application {
+        mainClass = "Main_ios.kt"
+        bundleID = "com.example-company.example-app"
+        version = "1.2.3"
```

Comment:
Desktop JVM dsl provides `packageVersion` for setting `CFBundleVersion` and `packageBuildString` for setting `CFBundleShortVersionString` https://github.com/JetBrains/compose-jb/tree/master/tutorials/Native_distributions_and_local_execution#specifying-package-version

Does it make sense to have similar properties for iOS?